### ISSUE TITLE
table already exists

### ISSUE THREAD

**Issue by brezenix** [coleifer/sqlite-browser#3]
Hi!
When I click on table name I get this error
```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python2.7/dist-packages/sqlite_browser/sqlite_browser.py", line 136, in inner
    return fn(table, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/sqlite_browser/sqlite_browser.py", line 152, in table_structure
    ds_table = dataset[table]
  File "/usr/local/lib/python2.7/dist-packages/playhouse/dataset.py", line 59, in __getitem__
    return Table(self, table, self._models.get(table))
  File "/usr/local/lib/python2.7/dist-packages/playhouse/dataset.py", line 147, in __init__
    model_class.create_table()
  File "/usr/local/lib/python2.7/dist-packages/peewee.py", line 3798, in create_table
    db.create_table(cls)
  File "/usr/local/lib/python2.7/dist-packages/peewee.py", line 2956, in create_table
    return self.execute_sql(*qc.create_table(model_class, safe))
  File "/usr/local/lib/python2.7/dist-packages/peewee.py", line 2877, in execute_sql
    self.commit()
  File "/usr/local/lib/python2.7/dist-packages/peewee.py", line 2732, in __exit__
    reraise(new_type, new_type(*exc_value.args), traceback)
  File "/usr/local/lib/python2.7/dist-packages/peewee.py", line 2869, in execute_sql
    cursor.execute(sql, params or ())
OperationalError: table "bondclear" already exists
```

**Comment by coleifer** [comment]
Do you mind sharing your actual database schema (just the `bondclear` table)?

**Comment by brezenix** [comment]
Yes, but this behavior is not dependent of table and reproduced on everyone.
```
CREATE TABLE BondClear( idInterClear INT NOT NULL, price DECIMAL(18,8) NOT NULL, idBond INT NOT NULL, idClearingSession INT NOT NULL, exportNum INT NOT NULL, PRIMARY KEY (idInterClear, idBond, idClearingSession, exportNum) )
```

**Comment by coleifer** [comment]
Interesting. I believe there was a bug in peewee that was fixed in 2.4.5:

https://github.com/coleifer/peewee/commit/e4917e964e0a4b5bc2a6da13e9afaeb97d12bf3b

Perhaps you could double-check what version of peewee you are using?

**Comment by coleifer** [comment]
Actually scratch that, I was able to replicate the issue with current versions.

**Comment by coleifer** [comment]
This is now fixed in coleifer/peewee@4ab0af417184aacd5964ede58d8174f8b931d28b

**Comment by brezenix** [comment]
You are right. The reason was in version of Peewee, but the version was 2.4.7
On other server where sqlite_browser works without errors the version of Peewee is 2.4.2
I have installed 2.4.2 on the server where I had got this error and now it is works fine :)
Thank you!
